html-templates/subtemplates/forms.tpl

```diff
@@ -69,3 +69,35 @@
 
 {template loginField}{field name=_LOGIN[username] label=Username required=true attribs='autofocus autocapitalize="none" autocorrect="off"' hint='You can also log in with your email address.'}{/template}
 {template passwordField}{field name=_LOGIN[password] label=Password hint='<a href="/register/recover">Forgot?</a>' required=true refill=false type=password}{/template}
+
+{template date inputName label='' error='' hint='' required=false autofocus=false attribs='' default=null class=null fieldClass=null max=null min=null}
```

Comment:
@mijogu this new template isn't needed, you can use the generic `{field}` template with `type` set for any of the enhanced input types, e.g. `{field type=date ...}`. The extra `date-field` CSS class isn't needed as CSS rules can select for the `type`